video/out/wayland_common.c

```diff
@@ -2744,6 +2850,20 @@ static void toggle_fullscreen(struct vo_wayland_state *wl)
     }
 }
 
+static void toggle_ime(struct vo_wayland_state *wl)
+{
+    struct vo_wayland_seat *seat;
+    wl_list_for_each(seat, &wl->seat_list, link) {
+        struct vo_wayland_text_input *ti = seat->text_input;
+        if (!ti)
+            continue;
+        if (wl->opts->wl_ime)
+            enable_ime(ti);
+        else
+            disable_ime(ti);
```

Comment:
```suggestion
        if (wl->opts->wl_ime) {
            enable_ime(ti);
        } else {
            disable_ime(ti);
        }
```
The code style is to use braces anyway on one line `if/else` like these

Reply:
Oops, forgot about that rule. Fixed.